builtins/amp-img.js

```diff
@@ -39,6 +39,15 @@ export class AmpImg extends BaseElement {
     this.srcset_ = null;
   }
 
+  srcChanged(newSrc) {
```

Comment:
So, this is a pretty critical part of this framework and ideally we'd find a very good solution:

1. The "attributeChanged" form might have issues with code obfuscation.
2. We need to see how make it convenient and simple to synchronize attribute updates with the lifecycle of the element. E.g. in the case of image, we have:

```
srcChanged(newSrc) {
  this.internalSrc = newSrc;
  if (this.isAfterLayout()) {
    vsync.mutate(() => this.img.src = this.internalSrc);
  }
}
```

Or something like this. Depending on a set of attributes, this could quickly become very entangled.